third_party/conditions/apis/conditions/v1alpha1/constants.go

```diff
@@ -0,0 +1,253 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+// ANCHOR: CommonConditions
```

Comment:
Do we use anything from this file?

Reply:
Some of the helpers in getter.go do. I wanted to do as little surgery as possible so that a future common library would be a drop-in replacement.